prime-router/docs/openapi.yml

```diff
@@ -726,6 +726,54 @@ components:
           description: The reporting facility id type (if different from OID)
           type: string
           default: null
+        suppressQstForAoe:
```

Comment:
The OpenAPI.yml document has fallen behind. Updated to match the current Hl7Configuration.